lib/chiavdf/fast_vdf/vdf_client.cpp

```diff
@@ -6,8 +6,10 @@ const int max_length = 2048;
 const int kMaxProcessesAllowed = 3;
 std::mutex socket_mutex;
 
+int process_number;
+
 void PrintInfo(std::string input) {
-    print("VDF Server: " + input);
+    print("VDF Client " + to_string(process_number) + ": " + input);
 }
 
```

Comment:
I have changed this file significantly in the integration branch (to reduce the boost dependency), I guess we will deal with that later.

Reply:
I'll try to combine your changes in integration and this PR in my new branch, new_vdf, where I implement the new VDF algorithm.